src/d3a/models/strategy/load_hours.py

```diff
@@ -58,8 +58,8 @@ def __init__(self, avg_power_W, hrs_per_day=None, hrs_of_day=None,
         if use_market_maker_rate:
             final_buying_rate = GlobalConfig.market_maker_rate
 
-        if isinstance(update_interval, int):
-            update_interval = duration(minutes=update_interval)
+        if isinstance(update_interval, (int, float)):
```

Comment:
Hmm, why are you allowing floats first but converting them to int anyways?

Reply:
I assume this is because the 'minutes' field probably does not accept floats. My question is, where does the float come from? Update interval units in d3a-web is minutes-int, right? If yes, why do you even need to check it with the isinstance here. If not, where is it generated from? It would be much better to cast it to duration at the point where this variable is converted to float, than cast it in d3a.

Reply:
You are right, I was being extra cautious. Reverting it